Src/IronPythonTest/Cases/IronPythonCasesManifest.ini

```diff
@@ -1,48 +1,28 @@
 ﻿[DEFAULT]
 Ignore=false
-IsolationLevel=ENGINE
+IsolationLevel=PROCESS
 WorkingDirectory=${ROOT}/Tests
 
 [test_cliclass]
-IsolationLevel=PROCESS
 Condition='$(OS)' != 'Unix'
 Reason=Causes an abort on mono, needs debug
 
-[test_datetime]
-IsolationLevel=ENGINE
-
-[test_dictcomp]
-IsolationLevel=PROCESS
-
 [test_dllsite]
 Ignore=true
 Reason=disabled in IronLanguages/main, needs lots of work
 
-[test_doc]
-IsolationLevel=PROCESS
```

Comment:
Maybe we should leave these in here in case we decide to revert the default at some point?